lite2/verifier.go

```diff
@@ -49,7 +49,7 @@ func Verify(
 	}
 
 	if h2.Height == h1.Height+1 {
-		if !bytes.Equal(h2.ValidatorsHash, h1NextVals.Hash()) {
+		if !bytes.Equal(h2.ValidatorsHash, h1NextVals.Hash()) { //h1.nextValidatorHash
```

Comment:
???

Reply:
In the spec they use h1.nextValidatorHash instead of h1NextVals.Hash. I have actually made the changes in another pull request